services/QuillLMS/app/models/stripe_checkout_session.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
```

Comment:
Should we have the auto-generated annotation code here from running `rake db:migrate`?